src/Microsoft.Identity.Web/Constants/IDWebErrorMessage.cs

```diff
@@ -32,6 +32,7 @@ internal static class IDWebErrorMessage
         public const string IssuerMetadataUrlIsRequired = "IDW10301: Azure AD Issuer metadata address URL is required. ";
         public const string NoMetadataDocumentRetrieverProvided = "IDW10302: No metadata document retriever is provided. ";
         public const string IssuerDoesNotMatchValidIssuers = "IDW10303: Issuer: '{0}', does not match any of the valid issuers provided for this application. ";
+        public const string B2CTfpIssuerNotSupported = "IDW10304: B2C issuer with 'tfp' in the URI is not supported. See https://aka.ms/ms-id-web/b2c-issuer for details.";
```

Comment:
maybe something like "microsoft identity web does not support a B2C issuer with...."